CondTools/SiPixel/test/SiPixel2DTemplateDBObjectUploader_cfg.py

```diff
@@ -79,7 +79,6 @@
 	quit()
 
 MagFieldValue = 10.*options.MagField #code needs it in deciTesla
-print '\nMagField = %f deciTesla \n'%(MagFieldValue)
```

Comment:
why removing this?

Reply:
Same thing as
https://github.com/cms-sw/cmssw/pull/31009#discussion_r464545505